lib/active_merchant/billing/credit_card.rb

```diff
@@ -158,6 +158,21 @@ def brand=(value)
       # @return [String]
       attr_accessor :fallback_reason
 
+      # Returns or sets whether card-present card data has been read contactlessly.
+      #
+      # @return [true, false]
+      attr_accessor :contactless
+
+      # Returns the ciphertext of the card's encrypted PIN.
+      #
+      # @return [String]
+      attr_accessor :encrypted_pin
+
+      # Returns the Key Serial Number (KSN) of the card's encrypted PIN.
+      #
+      # @return [String]
+      attr_accessor :encrypted_pin_key_id
```

Comment:
KSN has 3 parts: BDK ID, Reader Serial Number and Key ID.

It's the Reader that encrypting the PIN, not the Card.

encrypted_pin_key_id is the terminology used by Stripe so I assume it's what First Data is calling it too.